mlflow/keras.py

```diff
@@ -116,6 +118,24 @@ def save_model(keras_model, path, conda_env=None, mlflow_model=Model(), custom_o
                          attempt to infer the Keras module based on the given model.
     :param kwargs: kwargs to pass to ``keras_model.save`` method.
 
+    :param signature: (Experimental) :py:class:`ModelSignature <mlflow.models.ModelSignature>`
```

Comment:
No action needed, just confirmed that `signature` and `input_example` are not already arguments to [`keras_model.save`](https://www.tensorflow.org/api_docs/python/tf/keras/Model#save) - there's actually an argument called `signatures`, but no `signature` :P